lib/commands/audit.js

```diff
@@ -188,19 +190,41 @@ class VerifySignatures {
   }
 
   async setKeys ({ registry }) {
-    const keys = await fetch.json('/-/npm/v1/keys', {
-      ...this.npm.flatOptions,
-      registry,
-    }).then(({ keys: ks }) => ks.map((key) => ({
-      ...key,
-      pemkey: `-----BEGIN PUBLIC KEY-----\n${key.key}\n-----END PUBLIC KEY-----`,
-    }))).catch(err => {
-      if (err.code === 'E404' || err.code === 'E400') {
-        return null
-      } else {
-        throw err
-      }
-    })
+    const { host, pathname } = new URL(registry)
+    const regKey = `${host}${pathname === '/' ? '' : pathname}/keys.json`
+    const tufCachePath = path.join(this.npm.cache, '_tuf')
```

Comment:
Oh nice, I guess this means we get a clean cache every test run? I wonder, would we want to make the same change to `pacote` so it re-uses the same cache?